INSTALL.md

```diff
@@ -39,6 +39,20 @@ Using the brew package manager, simply run:
 
     brew install git-crypt
 
+Red Hat Enterprise Linux
+------------------------
+
+
+    sudo su
```

Comment:
I would personally say it'd look better as
$ sudo yum install git openssl openssl-devel